### DISCUSSION TITLE
Class "NumberFormatter" not found

### DISCUSSION THREAD

**Discussion by Shipu** [NativePHP/laravel#174] [Q&A] [answered]
is there any way to enable `intl` extension.

**Comment by osc2nuke** [comment]
It is not enabled in php.ini (Nativephp v0.3.3)
Search:
`;extension=intl`
remove:
`;`
Save the php.ini and reload the app.

Tested in a Blade view:
```
@php
   $a = new NumberFormatter("it-IT", \NumberFormatter::CURRENCY);

echo $a->formatCurrency(12345, "USD") . "<br>"; // outputs $ 12.345,00 and it is formatted using the italian notation (comma as decimal separator)

@endphp
```
Output:
`12.345,00 USD`

**Reply by emsifa** [reply]
Where is `php.ini` file?

**Reply by bomshteyn** [reply]
Is it possible to overrule this by using the `phpIni()` method in the `NativeAppServiceProvider`? If yes how?

I tried 
```PHP
public function phpIni(): array
{
    return [
        'extension'=>'intl'
    ];
}
```

But still got `Class "NumberFormatter" not found` once i hit the page using it

**Reply by osc2nuke** [reply]
That's what i also tought, but it does not do that.

**Comment by simonhamp** [comment, marked as answer]
Duplicate of #292
